### DISCUSSION TITLE
UNC path visible in SEB filebrowser

### DISCUSSION THREAD

**Discussion by johan-werk** [SafeExamBrowser/seb-win-refactoring#430] [Q&A]
Hello,

Is it possible to visible the UNC path in the SEB filebrowser (see picture)?
We want that students doesn't knows this UNC path in case of privacy

![WhatsApp Image 2022-06-27 at 2 18 53 PM](https://user-images.githubusercontent.com/108275705/175940642-01430121-1396-43f9-9766-b15095b430b4.jpeg)
e

**Comment by dbuechel** [comment]
If I understand you correctly, you'd like to hide the absolute path rendered below the file system tree? If yes, then that's at the moment unfortunately not possible, but please feel free to create a new feature request under [Issues](https://github.com/SafeExamBrowser/seb-win-refactoring/issues).

**Reply by johan-werk** [reply]
Yes, you understand me correctly.
